digdag-core/src/test/java/io/digdag/core/workflow/ForEachOperatorFactory.java

```diff
@@ -41,17 +45,19 @@ public String getType()
     @Override
     public ForEachOperator newOperator(OperatorContext context)
     {
-        return new ForEachOperator(context);
+        return new ForEachOperator(context, limits);
     }
 
     static class ForEachOperator
             implements Operator
     {
         private final TaskRequest request;
+        private final Limits limits;
 
-        public ForEachOperator(OperatorContext context)
+        public ForEachOperator(OperatorContext context, Limits limits)
```

Comment:
Is it possible to extend OperatorContext to have `limits` field instead?

Reply:
Good point. But OperatorContext is defined in digdag-spi and Limits is defined in digdag-core. So we cannot directly reference Limits from OperatorContext.
We may need to find good way to fix it.

Reply:
I added a method `getMaxWorkflowTasks()` in `OperatorContext` and set from Limits instance.